internal/scheduler/job/query.go

```diff
@@ -74,17 +74,10 @@ const updateProgressQuery = `
 `
 
 const completeRunQuery = `
-	update
-	  job_run
-	set
-	  completed_count = ?,
-	  total_count     = ?,
-	  retries_count   = ?,
-	  status          = 'completed',
-	  end_time        = current_timestamp
+	delete from job_run
 	where
-	  private_id = ?
-	  and status = 'running'
+		private_id = ?
```

Comment:
We normally apply our sql style [guide](https://github.com/hashicorp/boundary/blob/main/sql-style-guide.md#formatting) to `query.go` files so thses should not be tabs

Reply:
Addressed in https://github.com/hashicorp/boundary/pull/5204/commits/27c432a74853cad9c69e4d4f44fb87d9cadb9ec7